test/unit/test_parsing_error.py

```diff
@@ -0,0 +1,17 @@
+from scripts.test.shared import WASM_OPT, run_process
+from utils import BinaryenTestCase
+import os
+
+
+class ParsingErrorTest(BinaryenTestCase):
+  def test_parsing_error_msg(self):
+    module = '''(module
```

Comment:
I've been formatting these to line up with the surrounding Python code so they don't stick out so much. For example,
```suggestion
    module = '''
    (module
     (func $foo
      (abc)
     )
    )
    '''
Of course this has the downside of changing the contents of the string, but I think that should be fine for this test. 

```

Reply:
Oh this feature is cool! How can I suggest a change?

Reply:
The leftmost icon above the text box. Unfortunately I seem to have screwed this one up.

Reply:
Oh but your comment got committed together 😂